opensearchpy/connection/http_async.py

```diff
@@ -7,6 +7,23 @@
 # Modifications Copyright OpenSearch Contributors. See
 # GitHub history for details.
 #
+#  Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
These licenses headers are not required because this file was created post fork.

Reply:
Made the change suggested.